src/lib/detector.test.ts

```diff
@@ -13,56 +13,14 @@
 */
 /**
  * @file lib/detector.test.ts
- * @author Hugo Masclet <@Hugoo>
```

Comment:
Please keep

Reply:
as above, non of your/our your code is left in this file after removing this function.